parlai/crowdsourcing/utils/tests.py

```diff
@@ -135,22 +153,28 @@ def _register_mock_agents(
 
         for idx in range(num_agents):
 
-            mock_worker_name = f"MOCK_WORKER_{idx:d}"
-            max_num_tries = 6
+            mock_worker_registration_name = f"MOCK_WORKER_{idx:d}"
+            mock_worker_name = f"{mock_worker_registration_name}_sandbox"
+            max_num_tries = 3
```

Comment:
As an aside, retries should no longer be necessary with `assert_sandbox_worker_created` and `await_channel_requests` which run the async loop until pending things are processed.

Reply:
Just removed this retry loop without breaking the tests